app/ui-utils/client/lib/reactionListContent.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box, Tag, Divider } from '@rocket.chat/fuselage';
+
+import { useTranslation } from '../../../../client/contexts/TranslationContext';
+import Emoji from '../../../../client/components/basic/Emoji';
+
+export function Reactions(props) {
+	const styleDivider = {
+		marginTop: '8px',
+		marginBottom: '16px',
+	};
+
+	const reactionKeys = Object.keys(props.reactions);
```

Comment:
We can cache this value if the object at `props.reactions` remains unchanged in consecutive updates:

```suggestion
	const reactionKeys = useMemo(() => Object.keys(props.reactions), [props.reactions]);
```